easybuild/easyconfigs/s/SVclone/SVclone-1.1.2-foss-2022b.eb

```diff
@@ -0,0 +1,47 @@
+# EasyBuild EasyConfig
+#
+#  Fred Hutch Cancer Center - Seattle Washington US
+#
+#  John Dey
+easyblock = 'PythonBundle'
+
+name = 'SVclone'
+version = '1.1.2'
+
+homepage = 'https://github.com/mcmero/SVclone'
+description = """Cluster structural variants of similar cancer cell fraction (CCF)."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pysam', '0.19.1', {
+        'checksums': ['dee403cbdf232170c1e11cc24c76e7dd748fc672ad38eb0414f3b9d569b1448f'],
+    }),
+    ('PyVCF3', '1.0.3', {
+        'modulename': 'vcf',
+        'checksums': ['4b16d71c8b97010487e2c939fb4d5707b7bbfa4e2b313df9dba3e372c5ba031d'],
+    }),
+    (name, version, {
+        'modulename': False,
```

Comment:
Haven't seen this usage before - could you explain @fizwit (as much for my benefit as anything else).

Reply:
Are you referring to the 'modulename': False?  At first it seemed that SVclone only installed to /bin, But know I see a python library and `import SVclone` works fine.   I will update the PR.

Reply:
FYI, Easybuild sanity check imports module names as %(namelower)s.  Need to implicitly use `'modulename': '%(name)s',` to pass sanity check.

Reply:
no need to use the `'%(name)s'` template, can use `name` directly, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/18084/commits/e1aa6c6e7102578e8ff4456d2a089b9cc122c3f3